### PR TITLE
Customize app launcher drawer portlet

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue [hunk 1]
```diff
@@ -33,18 +33,29 @@
...
                 <v-divider
                    :inset="inset"
-                   class="my-0"/>
+                   class="my-0 appHeaderBorder"/>
 
                 <v-row class="mx-0 px-3">
                     <div class="appLauncherList">
                         <div class="appLauncherItem" v-for="(application, index) in favoriteApplicationsList">
-                            <a target="_blank" :href="application.appUrl">
+                            <a v-if="index < 9 " target="_blank" :href="application.appUrl">
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: why index < 9 here ?

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue [hunk 2]
```diff
@@ -33,18 +33,29 @@
...
                 </v-row>
+                <v-row class="seeAllApplications mx-0">
+                    <v-card
+                            flat
+                            tile
+                            class="d-flex flex justify-center mx-2">
+                        <a
+                                class="primary--text seeAllApplicationsBtn"
+                                href="/portal/intranet/appCenterUserSetup"
+                                >View all applications</a>
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: This label should internationalized

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: app-center-webapps/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 3]
```diff
@@ -118,4 +118,16 @@
     </depends>
   </module>
 
+  <module>
+    <name>appCenterBundle</name>
+    <script>
+      <minify>false</minify>
+      <path>/javascript/App-center.js</path>
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/WEB-INF/gatein-resources.xml]:
```suggestion
      <path>/javascript/appCenter.js</path>
```

**Review by azayati — COMMENTED** [review]
See review

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue [hunk 4]
```diff
@@ -77,7 +91,7 @@
                 }).then(data => {
                     this.favoriteApplicationsList = data.applications;
                 })
-            }
+            },
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: No needed I think

**Review by azayati — COMMENTED** [review]
See review

File: app-center-webapps/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 3 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/WEB-INF/gatein-resources.xml]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue (hunk 4 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/appLauncher/components/AppLauncher.vue]: Okey

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**